operator/tests/e2e-with-flags/nodepools/05-clean.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: kuttl.dev/v1beta1
```

Comment:
NIT: The number `05` should be `03` in the fole name. That would indicate the step within test case.